### PR TITLE
Edits from @jessegmeyerlab 

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Here are the in-line changes from @jessegmeyerlab. Comments that weren't a straight-forward fix are entered as comments. These are great edits, thank you so much for the thorough review!

### Suggested reviewers (optional)

@jessegmeyerlab 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -309,16 +317,16 @@ Tests that identify SARS-CoV-2 using nucleic-acid-based technologies will identi
...
 
 Additionally, the emerging nature of the COVID-19 pandemic has introduced some challenges related to uncertainty surrounding interactions between SARS-CoV-2 and its human hosts.
 For example, viral shedding kinetics are still not well understood but are expected to introduce a significant effect of timing of sample collection on test results [@doi:10.1038/d41587-020-00002-2].
 Similarly, the type of specimen could also influence outcomes, as success in viral detection varies among clinical sample types [@doi:10.1038/d41587-020-00002-2; @doi:10/gjrfns; @doi:10.2807/1560-7917.ES.2020.25.50.2000568].
-With CRISPR-based testing strategies, the gRNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur.
+With CRISPR-based testing strategies, the gRNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur.<!--Clarification: I don’t think this should be “patient’s genome” because the virus doesn’t integrate into human genomes?-->
```
Comment (rando2) [inline content/10.diagnostics.md:329]: @SystemsResearch @nilswellhausen would either of you be able to confirm whether this line is what was intended?

File: content/10.diagnostics.md [hunk 2]
```diff
@@ -350,13 +356,13 @@ IgG titers peaked by day 60, and persisted in all donors through the two-year du
...
-This information can be useful for understanding the extent to which a viral threat has affected a particular population.
 Several technologies have been used to developed serological tests for COVID-19, including ELISA, lateral flow immunoassay, chemiluminescence immunoassay, and neutralizing antibody assays [@doi:10.1093/ajcp/aqaa112].
 
 #### ELISA
 
-The application of ELISA to serological testing is complementary to its use in molecular diagnostics (above).
-Instead of using an enzyme-labeled antibody as a probe that binds to the target antigen, instead the probe is an antigen and the target is an antibody.
+The application of ELISA to serological testing is complementary to its use in molecular diagnostics (described above).
+Instead of using an enzyme-labeled antibody as a probe that binds to the target antigen, the probe is an antigen and the target is an antibody.
+The enzyme used for detection and signal amplification is on a secondary antibody raised generally against human IgG or IgM.<!--Request review on this sentence-->
```
Comment (rando2) [inline content/10.diagnostics.md:365]: @cbrueffer @nilswellhausen @RLordan Is anyone able to review this sentence? (review requested by the author, @jessegmeyerlab)

File: content/10.diagnostics.md [hunk 3]
```diff
@@ -495,26 +503,28 @@ This approach can utilize several of the technologies described for molecular te
...
 This corresponded to ~1.5% of the population, but ~12% of individuals sampled statewide were estimated as positive through antibody tests (along with indications of spatial heterogeneity at higher resolution) [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
 Technological advancements that facilitate widespread, rapid testing would therefore be valuable for accurately assessing the rate of infection and aid in controlling the virus' spread.
 Additionally, the trade off of accessibility, sensitivity, and time to results has raised some complex questions around which tests are best suited to certain situations.
-Immunoassays, including serological tests, have much lower limits of detection than PCR tests do [@doi:10.2142/biophysico.bppb-v18.004].
+Immunoassays, including serological tests, have much lower limits of detection than PCR tests do [@doi:10.2142/biophysico.bppb-v18.004].<!-- check accuracy, this sentence is likely flipped?-->
```
Comment (rando2) [inline content/10.diagnostics.md:514]: I wrote this and I definitely flipped the order!

```suggestion
Immunoassays, including serological tests, have much higher limits of detection than PCR tests do [@doi:10.2142/biophysico.bppb-v18.004].
```

File: content/10.diagnostics.md [hunk 4]
```diff
@@ -114,26 +114,26 @@ In dPCR, a sample is partitioned into thousands of compartments, such as nanodro
...
 This study analyzed 196 samples, including 103 samples from suspected patients, 77 from contacts and close contacts, and 16 from suspected convalescents, using both RT-qPCR and RT-ddPCR.
 Of the 103 suspected patient samples, RT-qPCR identified 29 as positive, 25 as negative, and 49 as suspected.
 Of the RT-qPCR negative or suspected samples, a total of 61 (19 negative and 42 suspected) were confirmed to be positive by RT-ddPCR.
 All 103 of the suspected patients were later confirmed to be SARS-CoV-2 positive through a combination of symptom development and RT-qPCR resampling, indicating that RT-ddPCR improved the overall detection rate among these patients from 28.2% to 87.4%.
 Of 77 patient samples from contacts and close contacts, 48 tested negative with both methods, and these patients were observed to remain healthy over a period of 14 days.
-Within the remaining 29 patient samples, 12 tested positive, 1 negative, and 16 suspected with RT-qPCR.
+Within the remaining 29 patient samples, 12 tested positive, 1 negative, and 16 suspected with RT-qPCR.<!--What does suspected mean hear?-->
```
Comment (rando2) [inline content/10.diagnostics.md:130]:
```suggestion
Within the remaining 29 patient samples, 12 were identified as positive, 1 as negative, and 16 as borderline with RT-qPCR.
```

File: content/10.diagnostics.md [hunk 5]
```diff
@@ -331,14 +339,12 @@ The rapid development of molecular tests has provided a valuable, albeit imperfe
...
-
+These tests are particularly useful for insight into population-level dynamics and can also offer a glimpse into the development of antibodies by individual patients during the course of a disease.
 Immunoassays can detect antibodies produced by the adaptive immune system in response to viral threat.
-In the process of mounting a response to a pathogen, the adaptive immune system produces antibodies specific to the pathogen.
 Understanding the acquisition and retention of antibodies is important both to the diagnosis of prior (inactive) infections and to the development of vaccines.
-The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies found in the blood.
+The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies found in the blood.<!--Needs Citation-->
```
Comment (rando2) [inline content/10.diagnostics.md:347]: Book is also added as a manual ref under URL, so see if this works. If it does, can remove.

```suggestion
The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies [@doi:10.3389/fimmu.2014.00520; @isbn:9780815336426].
```

File: content/10.diagnostics.md (hunk 2 above)
Comment (RLordan) [reply content/10.diagnostics.md:365]: I think this is correct + anti-human pan Ig as per: https://doi.org/10.1101/2020.04.24.057323

**Review by RLordan — APPROVED** [review]

File: content/10.diagnostics.md (hunk 1 above)
Comment (rando2) [reply content/10.diagnostics.md:329]: @jessegmeyerlab I found a reference that I think agrees with your concern. I propose this change:

```suggestion
With CRISPR-based testing strategies, the gRNA can recognize off-target interspersed sequences in the viral genome [@doi:10.3389/fcimb.2020.576875], potentially resulting in false positives and a loss of specificity.
```

Comment (jessegmeyerlab) [reply content/10.diagnostics.md:329]: That makes more sense thank you!

File: content/10.diagnostics.md [hunk 6]
```diff
@@ -331,14 +339,12 @@ The rapid development of molecular tests has provided a valuable, albeit imperfe
...
-
+These tests are particularly useful for insight into population-level dynamics and can also offer a glimpse into the development of antibodies by individual patients during the course of a disease.
 Immunoassays can detect antibodies produced by the adaptive immune system in response to viral threat.
-In the process of mounting a response to a pathogen, the adaptive immune system produces antibodies specific to the pathogen.
 Understanding the acquisition and retention of antibodies is important both to the diagnosis of prior (inactive) infections and to the development of vaccines.
-The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies found in the blood.
+The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies [@doi:10.3389/fimmu.2014.00520; @isbn:9780815336426].
```
Comment (rando2) [inline content/10.diagnostics.md:347]: We probably want to use the manual reference :laughing:

Comment (rando2) [reply content/10.diagnostics.md:347]:
```suggestion
The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies [@url:https://www.ncbi.nlm.nih.gov/books/NBK27162].
```

**Pull request merged**